manageiq-providers-azure.gemspec

```diff
@@ -13,6 +13,8 @@ Gem::Specification.new do |s|
 
   s.files = Dir["{app,config.lib}/**/*"]
 
+  s.add_dependency "azure-armrest", "~> 0.5.2"
+
   s.add_development_dependency "codeclimate-test-reporter", "~> 1.0.0"
```

Comment:
@djberg96 
For gems that are developed by ManageIQ we want to point to a specific gem version, so:"=0.5.2" instead of "~> 0.5.2". This means we can release a new version of a gem independent of upgrading manageiq to use it.